docs/reference/ml/ml-shared.asciidoc

```diff
@@ -537,24 +537,26 @@ the detectors in the `analysis_config`, starting at zero.
 end::detector-index[]
 
 tag::dfas-alpha[]
-Advanced configuration option. {ml-cap} uses loss guided tree growing.
-This means that trees will grow where the regularized loss reduces
-the most. This parameter multiplies a term based on tree depth in
-the regularized loss. Higher values result in shallower trees
-and faster training times. Values should be greater than or equal
-to zero. By default, this value is calculated during hyperparameter optimization.
+Advanced configuration option. {ml-cap} uses loss guided tree growing, which
+means that the decision trees grow where the regularized loss decreases most
+quickly. This parameter affects loss calculations by acting as a multiplier of 
+the tree depth. Higher alpha values result in shallower trees and faster 
+training times.
+//TBD: but potentially reduce accuracy?
```

Comment:
Unfortunately this is quite nuanced and very much data-dependent. I think we might be better off without mentioning anything more here for now. @valeriy42 is working on an example notebook that goes into more detail on explaining hyperparameters. When that is ready we can probably link to that for users that want to delve into the more nuanced details.

Reply:
Sounds good to me, thanks!  I'll remove that TBD